lib/ukdebug/gdbstub.c

```diff
@@ -1106,3 +1113,78 @@ int gdb_dbg_trap(int errnr, struct __regs *regs)
 
 	return r;
 }
+
+static int gdb_setup_backing_console(void)
+{
+	/* Ignore breakpoints if the stub is not enabled on the command line. */
+	if (gdb_backing_console_id == GDB_BACKING_CONSOLE_NONE) {
+		uk_pr_info("Backing console not set\n");
```

Comment:
I think we should not print in this case. If an image is compiled with GDB support but it is not enabled via command line, we should just silently continue.

Reply:
If you want to keep the message, you can use `uk_pr_debug`